### PR TITLE
 GameEngine.InputHandlerEvent added

### PR DESCRIPTION
**Pull request by fabeat**
See Amazon docs:
https://developer.amazon.com/de/docs/gadget-skills/gameengine-interface-reference.html#requests

### CODE REVIEW

**Comment by fabeat** [comment]
@maxbeckers to support PHPUnit 6, I cannot use `assertIsArray()`. I could use the deprecated `assertInternalType()` or `assertTrue(is_array())`. What shall it be? :)

**Comment by maxbeckers** [comment]
@fabeat thanks for the pr.
i think in this case we should use `assertTrue(is_array())`. And i'll have a look whats the Problem here.

File: src/Request/Request/GameEngine/Event/InputEvent.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     * @throws \Exception
+     *
+     * @return InputEvent
+     */
+    public static function fromAmazonRequest(array $amazonRequest): self
+    {
+        $event = new self();
+
+        $event->gadgetId   = $amazonRequest['gadgetId'];
+        $event->timestamp  = new \DateTime($amazonRequest['timestamp']);
```
Comment (maxbeckers) [inline src/Request/Request/GameEngine/Event/InputEvent.php]: Do we need here a solution like https://github.com/maxbeckers/amazon-alexa-php/blob/master/src/Request/Request/Standard/StandardRequest.php#L33 ? Maybe the dev console here also works not the expected way.

**Comment by fabeat** [comment]
> i think in this case we should use `assertTrue(is_array())`. And i'll have a look whats the Problem here.

I changed it to this syntax. The problem is that PHP 7.0 requires PHPUnit 6 which is no longer supported.

File: src/Request/Request/GameEngine/Event/InputEvent.php (hunk 1 above)
Comment (fabeat) [reply src/Request/Request/GameEngine/Event/InputEvent.php]: I tried it with the dev console. It returns the right timestamps. Could it be that this issue is resolved?

Comment (maxbeckers) [reply src/Request/Request/GameEngine/Event/InputEvent.php]: Yes may be. Or in this case it was still working.

**Comment by maxbeckers** [comment]
Yes i see the problem. So we take the way, the cs fixer wants to have. Sorry, my bad. For version 2.0 we will drop the php 7.0 Support. I still need to find some time for the updates.

**Pull request merged**